README.md

```diff
@@ -98,79 +98,50 @@ You can type ```/help``` to get details about what commands can be used, along w
 
 **Creating proposals:**
 
-Proposals can be drafted by utilizing various commands. 
-Upon completing a draft, it will automatically be posted so that you can view / make changes to it prior to publishing it.
+Creating a proposal requires careful consideration. Once you have launched a proposal it cannot be modified for transparency reasons. 
 
-When you decide to publish the draft, you can request the bot to do this.
+Start drafting your proposal in the #drafting-props channel and select the appropriate tag (budget, or governance). Depending on your tag selected the bot will handle the proposal prefix. Just create a name for your proposal.
 
-If you wish to change the emojis used when drafting & reaching soft consensus prior to publishing to snapshot this can be done in consts/constants.py. Simply replace YES_VOTE / NO_VOTE or ABSTAIN_VOTE with the desired emoji id. For example: 
+Please consider and use the following stuctures for the proposals body, for uniformity and structural reasons when making a proposal draft:
```

Comment:
"When making a draft, please use the following format for the proposal's body to ensure consistency across proposals:"